src/main/play-doc/operation/Install.md

```diff
@@ -1069,7 +938,7 @@ frontend monitor
 Run the HAProxy Docker container.
 
 ```
```

Comment:
Reused on the line https://github.com/typesafehub/conductr-doc/pull/256/files#diff-3f0467e546d1a3d3bad811c5e4d547abR940